cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/TransformerConversionTest.java

```diff
@@ -42,6 +42,58 @@ class TransformerConversionTest {
 
     private static final String EQUALS_LINE = "======================";
 
+    @Test
```

Comment:
What is the purpose of this file changes? Some methods were moved, but I don't understand why...